charts/greptimedb-cluster/values.yaml

```diff
@@ -1024,3 +1024,50 @@ grafana:
     type: ClusterIP
     # -- The annotations for the grafana service.
     annotations: {}
+
+# -- Deploy jaeger-all-in-one for development purpose.
+jaeger-all-in-one:
+  # -- Enable jaeger-all-in-one deployment.
+  enabled: false
+
```

Comment:
Add the following **jaeger-all-in-one** default values ​​here, reference: https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger#all-in-one-in-memory-configuration
```
provisionDataStore:
  cassandra: false
allInOne:
  enabled: true
storage:
  type: memory
agent:
  enabled: false
collector:
  enabled: false
query:
  enabled: false
```